src/main/kotlin/com/defenseunicorns/koci/Repository.kt

```diff
@@ -370,7 +370,7 @@ class Repository(
               }
             }
           }
-          .flattenMerge(concurrency = 3) // TODO: figure out best API to expose concurrency settings
+          .flattenMerge(concurrency = 6) // TODO: figure out best API to expose concurrency settings
```

Comment:
just a note - this slightly improves speed but also aligns this setting with Zarf's default OCI concurrency.